esp32_bluetooth_host/esp32_bluetooth_host.ino

```diff
@@ -93,6 +94,10 @@ class SettingsUpdatedCallback : public BLECharacteristicCallbacks {
     int reqFrequency = std::max(std::min(value, (uint16_t)MAX_SAMPLING_FREQUENCY), (uint16_t)MIN_SAMPLING_FREQUENCY);
     samplingDelay = round((float)TIMER_PRECISION / (float)reqFrequency);
     frequency = TIMER_PRECISION / samplingDelay;
+    Serial.print("Sampling delay:\t");
+    Serial.println(samplingDelay);
+    Serial.print("Sampling frequency:\t");
+    Serial.println(frequency);
```

Comment:
Maybe remove the debugging code, or make it more clear that it is to stay with a comment